llama_stack/apis/tools/tools.py

```diff
@@ -139,3 +172,14 @@ async def invoke_tool(
     ) -> ToolInvocationResult:
         """Run a tool with the given arguments"""
         ...
+
+
+# Three tool types:
```

Comment:
Would "memory" tools belong to 1 or 2?

Reply:
In this context, it will be built in, not to be confused with "builtin tool" for llama model. to avoid this confusion, i have changed the name to be ToolHost with three values:
```
@json_schema_type
class ToolHost(Enum):
    distribution = "distribution"
    client = "client"
    model_context_protocol = "model_context_protocol"

```